jellyfin-core/src/main/kotlin/org/jellyfin/sdk/Jellyfin.kt

```diff
@@ -20,12 +33,14 @@ public class Jellyfin(
 	 * Create a new API instance to use in API services.
 	 * The [clientInfo] and [deviceInfo] parameters are required when not passed as option in [JellyfinOptions].
 	 * The [baseUrl] is only required when HTTP calls are made.
+	 *
+	 * Throws an [IllegalStateException] when the client or device information is missing.
 	 */
 	public fun createApi(
 		baseUrl: String? = null,
 		accessToken: String? = null,
-		clientInfo: ClientInfo? = options.clientInfo,
-		deviceInfo: DeviceInfo? = options.deviceInfo
+		clientInfo: ClientInfo? = this.clientInfo,
+		deviceInfo: DeviceInfo? = this.deviceInfo,
```

Comment:
The `this` shouldn't be necessary I guess. Also, you could just leave this as-is, increases "performance" cause one less method call.

Reply:
The `this` is required because the parameter has the same name. I'll change it back to using `options` though.